config/jobs/kubernetes-csi/csi-driver-host-path/csi-driver-host-path-manual-job-config.yaml

```diff
@@ -47,17 +47,21 @@ presubmits:
           privileged: true
         resources:
           requests:
+            # these are both a bit below peak usage during build
+            # this is mostly for building kubernetes
+            memory: "9000Mi"
+            # during the tests more like 3-20m is used
             cpu: 2000m
 periodics:
 - interval: 6h
-  # 1.20 is used because we currently have pre-built images for it.
-  # The Kubernetes version does not matter much here, we just care
-  # about exercising distributed provisioning on some Kubernetes version.
+  # 1.21 is used because we the distributed deployment then also
```

Comment:
we the distributed deployment then also -> the distributed deployment also

Reply:
Fixed and pushed.